io/plugins/eu.esdihumboldt.hale.io.gml.geometry/src/eu/esdihumboldt/hale/io/gml/geometry/handler/PolygonHandler.java

```diff
@@ -160,10 +162,11 @@ public Object createGeometry(Instance instance, int srsDimension, IOProvider rea
 						// GeometryProperty<LinearRing> instance
 						GeometryProperty<LinearRing> ring = (GeometryProperty<LinearRing>) ((Instance) value)
 								.getValue();
-						outerRing.add(ring.getGeometry());
+						outer = ring.getGeometry();
```

Comment:
Did you encounter a specific problem with this? Because generally I think there should only be one outer ring, and in this case the behavior of both approaches should actually be the same. So did you encounter a case where there were multiple outer rings found here?